Tasks/HelmDeployV0/Tests/L0.ts

```diff
@@ -112,6 +114,27 @@ describe("HelmDeployV0 Suite", function () {
         done();
     });
 
+    it("Run successfully with Helm install (version 3) when invalid chart version is given", function(done: MochaDone) {        
```

Comment:
Are you ignoring the version if it's invalid?

Reply:
Yes. I'm showing a message that the version is invalid and running without the version flag.